src/app/dashboard-widgets/pipelines-widget/pipelines-widget.component.html

```diff
@@ -17,7 +19,10 @@ <h3>No Pipelines have run</h3>
         Pipelines run when you commit and build your code base.
       </p>
       <div class="f8-blank-slate-main-action">
-        <button id="spacehome-pipelines-add-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add to Space</button>
+      <div *ngIf="userOwnsSpace">
```

Comment:
PR https://github.com/fabric8-ui/fabric8-ui/pull/2898 uses ng-container instead of a div tag. I believe we should be consistent and use ng-container here. The extra div tags add to the DOM structure, which is unnecessary.

Reply:
Good catch.